docker/pom.xml

```diff
@@ -17,7 +17,7 @@
 	<properties>
 		<iaf.version>7.7-SNAPSHOT</iaf.version>
 		<revision>${iaf.version}</revision>
-		<h2.version>1.4.200</h2.version>
+		<h2.version>2.0.206</h2.version>
```

Comment:
Since this is used in 3 places, perhaps you can create a property in the parent pom.